src/core/storage/IndirectionVector.cpp

```diff
@@ -0,0 +1,14 @@
+#include <catch2/catch.hpp>
```

Comment:
Where this file is used, because it don't figure in the CmakeList

Reply:
I saw where it is used, but please consider to add it in the CMakeList

Reply:
It is and will remain unused. Removing it. Thanks.